python/python-video-generation/tutorial.md

```diff
@@ -0,0 +1,275 @@
+programming language used for data science, artificial intelligence and for
+web apps with [Flask][flask] or [Django][django]. However, Python is not only
+capable of achieving all of this, but here in Stack Builders we’ve also used
+it to create and render a fully customized video with animations, text and images using Python’s [MoviePy][moviepy], [SciPy][scipy] and [ImageMagick][imagemagick] libraries.
+
+## The Challenge
+
+Creating a customized video might sound easy, especially if tools for video
+editing are an option. Creating one or maybe a couple of them might even
+sound fun, but if you need to generate hundreds or even thousands of
+customized videos daily, this does not sound like a good idea. Imagine
+Facebook manually creating millions of videos for friendship anniversaries,
+birthdays and marketing campaigns for all their users. That would require an
+extreme amount of time to do, create delays in the delivery of the videos,
+and take away from an ideal user experience.
+
+Producing a video programmatically might not be a task developers come across
+daily. Having a firm understanding of what the final video should look like
+is vital for accomplishing this task.
+
+## Facing the challenge
+
+Here is where software development comes in handy and helps automate tasks to
+make our lives easier once again. Having a toolbelt filled with tools like
+Python gives us the upper hand in this challenge. This open source
+tutorial/contribution will explain how to work with multiple video segments
+using `MoviePy` tools together.
+
+## What is MoviePy?
+
+You might have heard of [FFMPEG][ffmpeg] or [ImageMagick][imagemagick] for
+image and video edition in a programmatically way. [MoviePy][moviepy] is a
+Python module for video editing (Python wrapper for FFMPEG and ImageMagick). It provides
+functions for cutting, concatenations, title insertions, video compositing,
+video processing, and the creation of custom effects. It can read and write
+common video and audio formats and be run on any platform with Python 2.7 or
+3+.
+
+## What is ImageMagick?
+
+[ImageMagick][imagemagick] is a open-source software suite, used for editing and manipulating digital images. It includes a command-line interface for executing complex image processing tasks, as well as APIs for integrating its features into software applications. `MoviePy` [TextClip][txt-clip] requires ImageMagick. 
+
+## What is SciPy?
+
+[SciPy][scipy] is a library used for scientific computing and technical computing. SciPy contains modules for image processing. MoviePy video tools plugins like [`Segmenting`][mpy-segmenting] benfits from this to locate objects and separate them.
+
+## What do we need?
+
+For this tutorial, is required ImageMagick installed and Python 3 with pipenv to manage its dependencies. All the
+modules can be installed from PyPI (Python Package Index) using `pip`. For
+further instructions check this file [here][readme].
```

Comment:
Since the blogpost content is not hosted in Github, we need the complete URL reference here